extensions/objects/templates/image-with-regions-editor.component.spec.ts

```diff
@@ -369,12 +369,12 @@ describe('ImageWithRegionsEditorComponent', () => {
       regionType: 'Rectangle',
       area: [
         [
-          0,
-          0
+          0.23225103021579202,
```

Comment:
why change these?

Reply:
I strongly think that these are the error in values that surfaced up after type checking of component file was done. Previous values placed were not correct. Some of the calculations were missing. See this comment https://github.com/oppia/oppia/pull/13532/files#r684461111